src/tests/ai_minimax_test.go

```diff
@@ -0,0 +1,20 @@
+package tests
+
+import (
+	"fmt"
+	"quarto/ai"
+	"quarto/state"
+	"testing"
+)
+
+func estInitAllTreeShouldReturnTree(t *testing.T) {
```

Comment:
Change `estInitAllTreeShouldReturnTree` to `TestInitAllTreeShouldReturnTree`

Reply:
s/estInitAllTreeShouldReturnTree/TestInitAllTreeShouldReturnTree/ ?

Reply:
Sedy is installed by default on all marmelab repositories :trophy: